lib/awful/mouse/snap.lua

```diff
@@ -67,7 +67,7 @@ end
 -- @param fixed_x True if the client isn't allowed to move in the x direction.
 -- @param fixed_y True if the client isn't allowed to move in the y direction.
 function module.snap(c, snap, x, y, fixed_x, fixed_y)
-    snap = snap or 8
+    snap = snap or snap.default_distance or 8
```

Comment:
And: Can't you move the `8` default value into an explicit setting of this? So somewhere here (next to the documentation of it) you do `module.default_distance = 8` and in this function you remove the `or 8`.